packages/iTwinUI-react/src/core/Wizard/Stepper.tsx

```diff
@@ -0,0 +1,104 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils';
+import '@itwin/itwinui-css/css/stepper.css';
+import { StepperStep } from './StepperStep';
+
+export type WizardLocalization = {
```

Comment:
Rename these to `Stepper*` and maybe do the same as I did in Modal: check ModalButtonBar. If not clear write a PM to me.

Reply:
Renamed to Stepper and added `WizardLocalization = StepperLocalization` inside of Wizard